data/presets/highway/hitchhiking.json

```diff
@@ -0,0 +1,19 @@
+{
+    "fields": [
+        "bench",
+        "bin",
+        "shelter"
+    ],
+    "geometry": [
+        "point"
+    ],
+    "tags": {
+        "highway": "hitchhiking"
```

Comment:
that really should have been `highway=signed_hitchhiking_place` to prevent degenerated use, but fixing it is out of scope for iD presets I guess

Reply:
Yeah, that will have to be a future discussion :-).
Will resolve this here …